### ISSUE TITLE
login issue

### ISSUE THREAD

**Issue by spiroid** [cozy-labs/cozy-docker#8]
Hi,

I rebuild the image on my laptop and i launched a container instance.
Then i successfully created an account with the init wizard, and i was redirected to the login page.

I was able to login with the newly created (green button) but right after i'm redirected to the login again.
And i can't pass this last step :(

How can i get more info on what's happening ?
Is this a known issue of one of the cozy component in a specific version ?

Thanks for your help.

**Comment by spiroid** [comment]
I had this error message reported in the proxy log file : 

```
[2015-07-31 17:47:53:731] error - app:error | Error: User already registered.
    at /usr/local/cozy/apps/proxy/build/server/controllers/authentication.js:56:17
    at /usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/lib/cozy_data_system.js:413:18
    at parseBody (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/main.js:26:12)
    at Request._callback (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/main.js:81:18)
    at Request.self.callback (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/node_modules/request/index.js:148:22)
    at Request.emit (events.js:98:17)
    at Request.<anonymous> (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/node_modules/request/index.js:876:14)
    at Request.emit (events.js:117:20)
    at IncomingMessage.<anonymous> (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/node_modules/request/index.js:827:12)
    at IncomingMessage.emit (events.js:117:20)
(END)
```

**Comment by clochix** [comment]
Thanks for reporting @spiroid :-)

Could you connect to a console inside the container and give us the result of `cozy-monitor versions` ?

Also, are you using some browser addon that may block cookies or something like that?

**Comment by spiroid** [comment]
@clochix: yep sure :)

```
$ cozy-monitor --version
1.2.27
```

```
$ cozy-monitor versions-stack

Cozy Stack:
controller: 2.0.29
data-system: 2.0.6
home: 2.0.6
proxy: 2.1.0
indexer: 1.0.6
monitor: 1.2.27
```

As of browser extensions, i had enabled the ghostery firefox extension.
I tried to make it work without this extension, and experienced the same issue.

hope this helps

**Comment by clochix** [comment]
Thanks @spiroid,
It appears that @Kloadut is experimenting the same behavior, so it will be easier for us to debug. Stay tuned ;-)

**Comment by spiroid** [comment]
up.
@clochix any progress on this issue ?

**Comment by Kloadut** [comment]
Hi

I just tested out following the README, and rebuilding the image from scratch:

```
sudo docker build -t cozy/full github.com/cozy-labs/cozy-docker
```

I was able to login after the onboarding wizard.

Are you sure that you are on the HTTPS `443` port ?

My Docker version is 1.7.1 by the way

**Comment by spiroid** [comment]
@Kloadut i cleaned my docker environment, removed cozy images and all orphans and triggered an new build. See my comment on issue https://github.com/cozy-labs/cozy-docker/issues/9#issuecomment-129781712.

After a new build from yesterday, it worked, i was able to finish the initialization wizard and i wasn't asked to log in as expected. Is the access restricted to HTTPS ?

I noticed that some components were updated, which might explain the difference:

```
$ cozy-monitor versions-stack

Cozy Stack:
controller: 2.0.30
data-system: 2.0.6
home: 2.0.7
proxy: 2.1.0
indexer: 1.0.6
monitor: 1.2.27
```

For the record, my docker version 1.6.2
